deb/scripts/_after-install.sh

```diff
@@ -2,4 +2,4 @@
 
 PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
 
-ldconfig
+ldconfig
```

Comment:
I prefer files to all end in a trailing newline, as is the standard for the linux kernel coding style (i'm not enforcing linux kernel coding style, but I try to be somewhat close to it)

I cleaned this up when I applied your patches for creating deb packages -- why did this change again in this patch series?